src/apps/vpn/platforms/ios/ioscontroller.swift

```diff
@@ -214,6 +214,10 @@ public class IOSControllerImpl : NSObject {
             Logger.global?.log(message: "Activating includeAllNetworks")
             proto!.includeAllNetworks = true
             proto!.excludeLocalNetworks = true
+
+            if #available(iOS 16.4, *) {
+                proto?.excludeAPNs = false
```

Comment:
1. proto! 
2. Can you write a comment about "false"? I would have written "true".

We want to exclude APNs so...why false?

Reply:
1. Why should we force unwrap here? I'd rather have us not set the flag than crash, if `proto` for some reason isn't found. Or gracefully handle the situation with a guard statement above. (I know it's a moot point here, as we force unwrap directly above this - but in general, I'd love to move towards having very few force unwraps in our code base.)

APNs exclusion or not is the big question of this PR, as this flag defaults to `true`. My thoughts: The VPN should capture as much traffic as possible. There may be some traffic we can't capture, and should be excluded. (I'm especially less worried about traffic going to the cell provider over the cellular network, as they already know your IP address and such.) In other words: My thought is that the default should be "all traffic in VPN", and we need a good reason as to why not to include something in the VPN. In this case, I can't find a reason to not include APNs traffic - It doesn't seem like including it has delivery reliability issues. In that case, why exclude it?

Reply:
1. I'm in agreement that there are far better ways to handle this. I'm personally from the "force unwraps should be super-rare in a code base" camp (though I've worked in codebases with other philosophies, and can adapt), and so I'm always reluctant to add a force unwrap to code That said, as we both noted this is made irrelevant by the force unwraps directy above. I fully agree this should be better handled (perhaps with a `guard` above), but that seems like a larger lift than this PR. I made this a force unwrap for now.
 
2. Good call - added that comment.